rust/migration/src/m20220101_000002_create_table.rs

```diff
@@ -101,6 +123,17 @@ impl MigrationTrait for Migration {
                             .to_col(Job::Id)
                             .on_delete(ForeignKeyAction::Cascade),
                     )
+                    .foreign_key(
+                        ForeignKeyCreateStatement::new()
+                            .name("fk-blob_id-blob")
+                            .from_tbl(OutputFile::Table)
+                            .from_col(OutputFile::BlobId)
+                            .to_tbl(Blob::Table)
+                            .to_col(Blob::Id)
+                            // A blob may not be deleted if there exists an
+                            // output file that depends on it.
+                            .on_delete(ForeignKeyAction::Restrict),
```

Comment:
ditto, for instance when we delete a job, we cascade the deletion to the output file